packages/slice-machine/src/components/Card/Card.tsx

```diff
@@ -136,18 +136,28 @@ export const CardActions: FC<PropsWithChildren> = (props) => (
 );
 
 type CardFooterProps = {
+  loading?: boolean;
+  startIcon?: IconName;
   action?: ReactNode;
   subtitle?: ReactNode;
   title?: ReactNode;
 };
 
 export const CardFooter: FC<CardFooterProps> = ({
   action,
+  loading = false,
+  startIcon,
   subtitle,
   title,
   ...otherProps
 }) => (
   <div {...otherProps} className={styles.footer}>
+    {(startIcon || loading) && (
+      <div className={styles.startIconBox}>
+        {loading && <ProgressCircle color="grey11" />}
+        {startIcon && <Icon name={startIcon} size="small" />}
```

Comment:
this can potentially show two elements on the screen, right? shouldn't it be
```suggestion
        {!loading && startIcon && <Icon name={startIcon} size="small" />}
```